Blocks/URL/test.cc

```diff
@@ -447,3 +447,63 @@ TEST(URLTest, FillsCurrentStructsFromURLParameters) {
     }
   }
 }
+
+TEST(URLTest, GetURLPathSeparator) {
+  EXPECT_EQ('/', current::url::GetURLPathSeparator());
```

Comment:
We're retiring this "getter". And there are always global `static_assert`-s on `constexpr`-s :)